bigframes/session/__init__.py

```diff
@@ -342,6 +342,37 @@ def read_gbq_query(
         ``row_number() over ()`` if there is no natural unique index or you
         want to preserve ordering.
 
+        **Examples:**
+
+            >>> import bigframes.pandas as bpd
+            >>> bpd.options.display.progress_bar = None
+
+            Preserve ordering in a query input.
+
+            >>> bpd.read_gbq_query('''
+            ...    SELECT
+            ...       -- Instead of an ORDER BY clause on the query, use
+            ...       -- ROW_NUMBER() to create an ordered DataFrame.
+            ...       ROW_NUMBER() OVER (ORDER BY AVG(pitchSpeed) DESC)
+            ...         AS rowindex,
+            ...
+            ...       pitcherFirstName,
+            ...       pitcherLastName,
+            ...       AVG(pitchSpeed) AS averagePitchSpeed
+            ...     FROM `bigquery-public-data.baseball.games_wide`
+            ...     WHERE year = 2016
+            ...     GROUP BY pitcherFirstName, pitcherLastName
+            ... ''', index_col="rowindex").head(n=5)
```

Comment:
Shall we split it into two steps?
df = ...
df.head(n=5)

This shows how users can define a dataframe